code/renderers/svelte/template/cli/js/Button.svelte

```diff
@@ -22,22 +22,13 @@
   $: mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
 
   $: style = backgroundColor ? `background-color: ${backgroundColor}` : '';
-
-  const dispatch = createEventDispatcher();
-
-  /**
-   * Optional click handler
-   */
-  export let onClick = (event) => {
-    dispatch('click', event);
-  };
 </script>
 
 <button
   type="button"
   class={['storybook-button', `storybook-button--${size}`, mode].join(' ')}
   {style}
-  on:click={onClick}
```

Comment:
Is this related to this PR?

Reply:
Made a new PR for this

Reply:
https://github.com/storybookjs/storybook/pull/20007